unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Interfaces/Extensions/InternalCollidersExtensions.cs

```diff
@@ -15,7 +15,7 @@ public static class InternalCollidersExtensions
             if (colliderInternalComponent.HasCollider(scene, entity, collider))
                 return;
 
-            var model = colliderInternalComponent.GetFor(scene, entity)?.model ?? new InternalColliders();
+            var model = colliderInternalComponent.GetFor(scene, entity)?.model ?? new InternalColliders(new KeyValueSet<Collider, uint>());
```

Comment:
same for this method - maybe we can cache, clear and re-use instead of creating new instances